Adding the Leaf, a MED Botany/Chemistry Ship for Medical Players

Rubeebeebee opened the pull request:
<!-- Guidelines: https://docs.spacestation14.io/en/getting-started/pr-guideline -->

## About the PR
<!-- What did you change? -->
<!-- If this is a code change, summarize at high level how your new code works. This makes it easier to review. -->
description: A retired balaamite warship, it is now living it's twilight years as a botany/chemistry ship. 
  price: 59000
  
  It's a nice little medship, meant to replace the hole that the eden left (rest in peace king)
  
  Its got plenty of space for growing crops for chemicals, medicine, food, etc. 
  and a fully stocked and equipped chemistry suite. 
  also features a kitchen to cook said crops into meals if desired. 
  
  Sadly, it doesnt have any medical capabilities, with it only being equipped with a medical techfab. 
  
## Why / Balance
<!-- Discuss how this would affect game balance or explain why it was changed. Link any relevant discussions or issues. -->
I made this to get the med players another ship to toy around with, now that the eden is gone. 

## How to test
<!-- Describe the way it can be tested -->
load up the .yml in an IDE or buy it from the CO CIV shipyard

## Media
<!-- Attach media if the PR makes ingame changes (clothing, items, features, etc). 
Small fixes/refactors are exempt. Media may be used in SS14 progress reports with credit. -->
<img width="898" height="893" alt="leaf profile live" src="https://github.com/user-attachments/assets/9e01c4ff-c477-42eb-919c-e8eab5ba2243" />
<img width="887" height="865" alt="leaf profile" src="https://github.com/user-attachments/assets/39f05ca6-35f8-4e8c-bc07-2610ca85c38f" />
<img width="814" height="238" alt="leaf appraisal" src="https://github.com/user-attachments/assets/6631e9d4-0b70-4e85-9009-04ba05a3910a" />

## Requirements
<!-- Confirm the following by placing an X in the brackets [X]: -->
- [X] I have read and am following the [Pull Request and Changelog Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html).
- [X] I have added media to this PR or it does not require an ingame showcase.
- [X] I can confirm this PR contains no AI-generated content, and did not use any AI-generated content.
<!-- You should understand that not following the above may get your PR closed at maintainer’s discretion -->

## Breaking changes
<!-- List any breaking changes, including namespaces, public class/method/field changes, prototype renames; and provide instructions for fixing them.
This will be posted in #codebase-changes. -->

**Changelog**
<!-- Add a Changelog entry to make players aware of new features or changes that could affect gameplay.
Make sure to read the guidelines and take this Changelog template out of the comment block in order for it to show up.
Changelog must have a :cl: symbol, so the bot recognizes the changes and adds them to the game's changelog. -->

:cl:
- add: the Leaf; a MED Botany/Chemistry ship to replace the Eden (rest in peace king)

Rubeebeebee committed:
adding the Leaf, a MED botany/chemistry ship for medical players

monolith8319 committed:
chore: automatically update REUSE headers

OneZerooo0 reviewed (changes requested):
touch up decals in chem/kitchen
add vacuum fix markers to frontal catwalks
increase price to like, 60-65k
missing holopad and a medical doctor locker

Rubeebeebee commented:
> touch up decals in chem/kitchen add vacuum fix markers to frontal catwalks increase price to like, 60-65k missing holopad and a medical doctor locker

gotcha, will add those!

Rubeebeebee committed:
updated the leaf to have a holopad + doc wall locker
increased price to 62k
added vacuum tiles to exterior catwalk
touched up decals

Rubeebeebee committed:
Merge branch 'Leaf-Branch' of https://github.com/Rubeebeebee/Monolith into Leaf-Branch

Rubeebeebee commented:
> touch up decals in chem/kitchen add vacuum fix markers to frontal catwalks increase price to like, 60-65k missing holopad and a medical doctor locker

okie dokie, ive done what you've asked!

also raised the price to 62k
<img width="875" height="872" alt="updated leaf" src="https://github.com/user-attachments/assets/66e4e2b8-89ce-4358-ba7c-b4cc398b9ab4" />

Rubeebeebee committed:
added 2x defbis and a wall recharger. I forgor :(

monolith8319 committed:
chore: automatically update REUSE headers

OneZerooo0 reviewed (changes requested):
defib/fire extinguisher?
Decals need more work, maybe touch up structure layout to help
Frontal airlocks are weird, also just not much in the frontal area that warrants its existence aside from the tree/religion thing
Overall just needs more work, just doesn't feel high enough quality

Rubeebeebee commented:
> defib/fire extinguisher? Decals need more work, maybe touch up structure layout to help Frontal airlocks are weird, also just not much in the frontal area that warrants its existence aside from the tree/religion thing Overall just needs more work, just doesn't feel high enough quality

yep, added a defib + wall extinguisher! i just uh... was too lazy to take pictures. [943b41d](https://github.com/Monolith-Station/Monolith/pull/1758/commits/943b41dedaad5a86f816fe7112eecfdfe0d10be7)

for the decals and the structure, what do you want to see? Can you give any examples?

and for the frontal structure, i suppose i can shorten it a little bit by removing the pew areas. It is supposed to be a retired balaamite batleship, and ive added the altar room there to show that little lore blurb

OneZerooo0 commented:
If it’s a retired battleship, it definitely doesn’t feel like one. Try making it darker, play around with the decals more I guess. More structural interest, front could have remnants of the original purpose, etc

Rubeebeebee committed:
updated it to be more retired battleship

Rubeebeebee committed:
Merge branch 'Leaf-Branch' of https://github.com/Rubeebeebee/Monolith into Leaf-Branch

Rubeebeebee commented:
> If it’s a retired battleship, it definitely doesn’t feel like one. Try making it darker, play around with the decals more I guess. More structural interest, front could have remnants of the original purpose, etc

right so, i took your advice and made it feel like a retired battleship! 
<img width="1057" height="693" alt="leafwar2 weapons" src="https://github.com/user-attachments/assets/858f485b-b752-4515-a3ed-9f18903f8889" />
I've added some frontal weapons + replaced the windows with actual frontal armor, and the side blast doors to show where the heavier guns wouldve been placed when it was still an active warship. 

I've also made the ship a little darker by removing all of the windows and replacing some of the lights with the dimmer fluorescent lights, as well as tweaking some of the decal and tile choices. 

<img width="888" height="862" alt="leafwar2 live" src="https://github.com/user-attachments/assets/ac64e6d2-da7a-4c87-bcfe-5a3aa61c48d3" />
<img width="872" height="835" alt="leafwar 2" src="https://github.com/user-attachments/assets/b5f20db0-129d-4dca-8078-fc714d4552ad" />

Also replaced 4x of the thursters at the back with a large thruster, and replaced the uranium generator with a bananium one to catch up with the energy load. Added a radiation suit so people can refuel safely.

Rubeebeebee commented:
> > If it’s a retired battleship, it definitely doesn’t feel like one. Try making it darker, play around with the decals more I guess. More structural interest, front could have remnants of the original purpose, etc
> 
> right so, i took your advice and made it feel like a retired battleship! <img alt="leafwar2 weapons" width="1057" height="693" src="https://private-user-images.githubusercontent.com/220646896/474302994-858f485b-b752-4515-a3ed-9f18903f8889.PNG?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NTQzNjQzMTcsIm5iZiI6MTc1NDM2NDAxNywicGF0aCI6Ii8yMjA2NDY4OTYvNDc0MzAyOTk0LTg1OGY0ODViLWI3NTItNDUxNS1hM2VkLTlmMTg5MDNmODg4OS5QTkc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUwODA1JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MDgwNVQwMzIwMTdaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0xYjhjNTgxNTliODQ3MmMxOGY4MjQwNWQxODFjN2QyYTVhNDdiZjE4ZDcyMTFkZTBjNzdiNmQ2MTAyYTE5YWJlJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.YMhv4E7KX0WbzV1qwvKSNFGTUW_SAq_U7344isvuJuo"> I've added some frontal weapons + replaced the windows with actual frontal armor, and the side blast doors to show where the heavier guns wouldve been placed when it was still an active warship.
> 
> I've also made the ship a little darker by removing all of the windows and replacing some of the lights with the dimmer fluorescent lights, as well as tweaking some of the decal and tile choices.
> <img alt="leafwar2 live" width="888" height="862" src="https://private-user-images.githubusercontent.com/220646896/474303236-ac64e6d2-da7a-4c87-bcfe-5a3aa61c48d3.PNG?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NTQzNjQzMTcsIm5iZiI6MTc1NDM2NDAxNywicGF0aCI6Ii8yMjA2NDY4OTYvNDc0MzAzMjM2LWFjNjRlNmQyLWRhN2EtNGM4Ny1iY2ZlLTVhM2FhNjFjNDhkMy5QTkc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUwODA1JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MDgwNVQwMzIwMTdaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT05ZTZmZDE0YzA5ZjJlN2NhY2Q3NmJhN2Q2YmQ3NDUyMzE2YWYwOTQyNmMxNDQ5NGI5NDJiMTExMDdlMGVlNGFhJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.YZPBVDLMa0HQsJU930e33uUxI6TGvv3eG_o1XhC6bRQ"> <img alt="leafwar 2" width="872" height="835" src="https://private-user-images.githubusercontent.com/220646896/474303242-b5f20db0-129d-4dca-8078-fc714d4552ad.PNG?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NTQzNjQzMTcsIm5iZiI6MTc1NDM2NDAxNywicGF0aCI6Ii8yMjA2NDY4OTYvNDc0MzAzMjQyLWI1ZjIwZGIwLTEyOWQtNGRjYS04MDc4LWZjNzE0ZDQ1NTJhZC5QTkc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUwODA1JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MDgwNVQwMzIwMTdaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0xMzYzZGFkYTI3ZWEwOTZhNzE5NTNmNmU2ZDdhNWRiYWFhMWM3YWJjNTBlMTUyZThmMGM0YjIyMWIyNjQyMDA3JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.hkR735KDLRhOW8_QLkOU0hyUsJ-imuMfWJRcRoMtAhE">
> 
> Also replaced 4x of the thursters at the back with a large thruster, and replaced the uranium generator with a bananium one to catch up with the energy load. Added a radiation suit so people can refuel safely.

Also updated the price to 72k to match the new additions, and tweaked the description to include the fact its hullers were carved out to make space for the botany/chemistry stuff

Rubeebeebee committed:
updated to ahve a roof component

Rubeebeebee commented:
updated to have a roof component

monolith8319 committed:
chore: automatically update REUSE headers

Rubeebeebee committed:
added roof tiles. Like a leaaaf in a canopy or something. im losing my mind.

Rubeebeebee committed:
Merge branch 'Leaf-Branch' of https://github.com/Rubeebeebee/Monolith into Leaf-Branch

Rubeebeebee committed:
updated to have a manufacturer prefix + removed balaamite reference

Rubeebeebee committed:
updated it with startch's recommendations

monolith8319 committed:
chore: automatically update REUSE headers

Rubeebeebee commented:
> missing vacuum markers on exterior catwalks front altar looks very shoved in, and dont premap faith implanters/bibles, the sodium/orange lighting is too bright imo too interior itself is very bland kitchen has pretty lazy decaling, and no sink engi needs more tile/decal work than just catwalks and the same maint tile chem tile/decalwork is generally low quality, medical techfab should be moved out of chem to somewhere else
<img width="870" height="856" alt="leaf updates" src="https://github.com/user-attachments/assets/3420e242-652f-454c-9ea8-0e0bac8a20a1" />

right so! I've fixed the issues you mentioned (although idk what you mean about exterior catwalks, they have vaccum tiles there) adjusted the lights to be a little dim (albiet the center room is bright since its supposed to be green-house like) removed the bible/shrine room entirely, engineering's a bit more decaled up i hope, chemistry decals also tweaked, and ive just straight up removed the kitchen to be a treatment room and moved the medical techfab there. 

added a manufacturing prefix (PTR = Peacetime retrofit), removed balaamite references, but idk what you mean about the generator type being wrong, its bananium.

Rubeebeebee commented:
@starch70 YOU'RE THE BEST WOOOOO!!!!
